.github/workflows/otbr.yml

```
@@ -108,11 +108,21 @@ jobs:
         include:
           - otbr_mdns: "mDNSResponder"
             otbr_trel: 0
+            cert_scripts: ./tests/scripts/thread-cert/border_router/*.py
+            description: ""
           - otbr_mdns: "mDNSResponder"
             otbr_trel: 1
+            cert_scripts: ./tests/scripts/thread-cert/border_router/*.py
+            description: ""
+          - otbr_mdns: "mDNSResponder"
+            otbr_trel: 0
+            cert_scripts: ./tests/scripts/thread-cert/border_router/MATN/*.py
+            description: "MATN"
           - otbr_mdns: "avahi"
             otbr_trel: 0
-    name: BR (${{ matrix.otbr_mdns }}, TREL=${{matrix.otbr_trel}})
+            cert_scripts: ./tests/scripts/thread-cert/border_router/*.py
+            description: ""
+    name: BR (${{ matrix.otbr_mdns }}, TREL=${{matrix.otbr_trel}}) ${{ matrix.description }}
```

```suggestion
    name: BR (${{ matrix.description }} ${{ matrix.otbr_mdns }}, TREL=${{matrix.otbr_trel}})
```
move description to be the first